connectors/src/connectors/zendesk/lib/permissions.ts

```diff
@@ -155,6 +144,19 @@ export async function retrieveChildrenNodes({
           };
           nodes.push(helpCenterNode);
         }
+        const ticketsNode: ContentNode = {
+          provider: connector.type,
+          internalId: getTicketsInternalId(connectorId, objectId),
+          parentInternalId: parentInternalId,
+          type: "folder",
+          title: `[${brandName}] Tickets`,
```

Comment:
we did not discuss these changes on how the content tree is rendered, we can see after but in the meantime please revert and keep this PR specific to tickets filtering 🙏

Reply:
That's a minor change, just adding the brand name on the content node - just good sense, because when we have several brands we see "help center" "help center" "tickets" "tickets" 

As it's minor, I'd propose keeping it there. This is a small PR, we can modify later but we already have Qonto that mentionned the issue (not sure why they already have access)

We can switch to another styling in a later PR

![Screenshot 2024-11-14 at 11 47 01](https://github.com/user-attachments/assets/2d964952-af51-4ff3-947b-069bd7175a8b)

Reply:
I'm stating that we should have discussed this because this is already handled in another PR that is awaiting review https://github.com/dust-tt/dust/pull/8552/files

Reply:
This PR achieves that without repeating the brand name in the `ConnectorPermissionsModal`
<img width="782" alt="Screenshot 2024-11-14 at 11 30 41 AM" src="https://github.com/user-attachments/assets/f6c599d6-2561-464e-934c-97f00c315f13">